adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -131,12 +131,24 @@ - (BOOL)addDelegate:(nonnull id<GADMAdapterVungleDelegate>)delegate {
     }
   } else if ([delegate respondsToSelector:@selector(isBannerAd)] && [delegate isBannerAd]) {
     @synchronized(_bannerDelegates) {
+      // We only support displaying one Vungle Banner Ad at the same time currently
+      if (_bannerPlacementID != nil && ![_bannerPlacementID isEqualToString:delegate.desiredPlacement]) {
+        return NO;
+      }
+
       if (delegate && ![_bannerDelegates objectForKey:delegate.bannerRequest]) {
         NSEnumerator *enumerator = _bannerDelegates.keyEnumerator;
         GADMAdapterVungleBannerRequest *bannerRequest = nil;
         while (bannerRequest = [enumerator nextObject]) {
           if ([bannerRequest.placementID isEqualToString:delegate.bannerRequest.placementID]) {
             return NO;
+          } else {
+            NSOrderedSet *set = [_bannerDelegates objectForKey:bannerRequest];
+            for(id<GADMAdapterVungleDelegate> delegate in set) {
+              if (delegate.bannerState == BannerRouterDelegateStateWillPlaying || delegate.bannerState == BannerRouterDelegateStatePlaying) {
+                return NO;
```

Comment:
When playing multiple Banners with different placementIds, we need to make sure only one Vungle Banner is displaying at the same time in multi threads environment.